README.md

```diff
@@ -20,6 +20,19 @@ __What features does it have?__
 - Responsive layout — works well on your smartphone
 - _.. and more!_
 
+__Why the fork?__
+
+We felt that the original [Shout](https://github.com/erming/shout) project
+"stagnated" a little because its original author wanted it to remain his pet
+project (which is a perfectly fine thing!).
+
+A bunch of people, excited about making things a bit differently than the upstream
+project forked it under a new name: «The Lounge».
+
+This forks aims to be community managed, meaning that the decisions are taken
+in a collegial fashion, and that a bunch of maintainers should be able to make
+the review process more streamlined.
```

Comment:
I would not use words like "should" or "ideally" here actually, but more like "goal(s)", "intent", more present tense. But maybe that doesn't make any sense and therefore you should ignore this comment :-)